pkg/minishift/hostfolder/hostfolder_manager.go

```diff
@@ -118,9 +119,9 @@ func (m *Manager) List(driver drivers.Driver) ([]MountInfo, error) {
 		case SSHFS.String():
 			source = hostFolder.Options[config.Source]
 		}
-
+		r := regexp.MustCompile(fmt.Sprintf("%s(\\s+)", hostFolder.MountPoint()))
 		mounted := false
-		if isRunning && strings.Contains(procMounts, hostFolder.MountPoint()) {
+		if isRunning && r.MatchString(procMounts) {
```

Comment:
We are doing the same thing as isHostFolderMounted() here. So why can not we reuse the function?  The only difference would be that we need to run more number of `cat /proc/mounts` commands but will that make a lot of difference. Also running same regular expression in two different functions is not a ideal thing to do. Ideally it should be done in one place.

Reply:
@praveenkumar I was thinking something like this

```
function doesMountMatch(string procMounts, string hostFolderMount) bool {
	r := regexp.MustCompile(fmt.Sprintf("%s(\\s+)", hostFolderMount)

	if !r.MatchString(procMounts) {
		return false 
	}
	return true
}
```